appinventor/components/src/com/google/appinventor/components/runtime/Form.java

```diff
@@ -1019,6 +1019,19 @@ public void VersionCode(int vCode) {
   public void VersionName(String vName) {
     // We don't actually need to do anything.
   }
+  
+  /**
+   * Specifies the App Name.
+   *
+   * @param aName the display name of the installed application in the phone
+   */
+  @DesignerProperty(editorType = PropertyTypeConstants.PROPERTY_TYPE_STRING,
+    defaultValue = "")
+  @SimpleProperty(userVisible = false,
```

Comment:
I'd expand this to add "if the App Name is blank, it will be set to the name of the project, when the project is built.